bundle/manifests/falcon-operator-manager-config_v1_configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: 70435a7a.crowdstrike.com
```

Comment:
Reading about electionLeader and resourceName, it appears `resourceName` is dynamically created during deployment. Should this be statically set here?

Reply:
All this code is generated from `make bundle`. Every time `make bundle` is run in the cli, it will change anything statically changed in `bundle/`. Really, this change needs to take place under `config/`. Stepping back though, the documentation from the Operator SDK standpoint isn't clear as far as handling this in the CRD. https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#leader-election talks at the golang level.

Reply:
I feel like at this point this should be dealt with when we start testing and are further a long down the development.